misc/pytorch_toolkit/chest_xray_screening_federated_gcn/src/utils/loss.py

```diff
@@ -0,0 +1,75 @@
+from torch import nn
+import torch
+import numpy as np
+
+############# Define the Weighted Loss. The weights are different for each class ########
+class Custom_Loss(nn.Module):
+    def __init__(self, site, device=torch.device('cpu')):
+        super(Custom_Loss, self).__init__()
+        if site==-999:
+            wts_pos = np.array([ 6.07201409, 12.57545272,  5.07639982,  1.29352719, 14.83679525,  
```

Comment:
Define these weights (lists given as input to np.array) in a JSON file and read them here.